frontend/fingertips-frontend/components/charts/Inequalities/Inequalities.test.tsx

```diff
@@ -87,11 +87,14 @@ describe('Inequalities', () => {
     'inequalitiesTypes-dropDown-component-lc',
   ])('should render inequalities component: %s', async (testId) => {
     await testRender(testData);
+    fireEvent.click(screen.getByText('Show inequalities data'));
```

Comment:
```suggestion
  await userEvent.click(screen.getByText('Show inequalities data'));
```
please change all fireEvents to userEvent, which is newer and is better test for what a user would do.